deployment/CP05MOAS-A6263_Deploy.csv

```diff
@@ -1,57 +1,78 @@
+WJ20190709,,WJ20190709,,CP05MOAS-A6263-00-ENG000000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGCON-AUVPNR-06263,40.01075,-70.567783,,,,along-shelf survey
+WJ20190709,,WJ20190709,,CP05MOAS-A6263-01-FLORTN000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-01108,40.01075,-70.567783,,,,along-shelf survey
+WJ20190709,,WJ20190709,,CP05MOAS-A6263-02-DOSTAN000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-01519,40.01075,-70.567783,,,,along-shelf survey
+WJ20190709,,WJ20190709,,CP05MOAS-A6263-03-CTDAVN000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00424,40.01075,-70.567783,,,,along-shelf survey
+WJ20190709,,WJ20190709,,CP05MOAS-A6263-04-NUTNRN000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-00366,40.01075,-70.567783,,,,along-shelf survey
+WJ20190709,,WJ20190709,,CP05MOAS-A6263-05-ADCPAN000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.01075,-70.567783,,,,along-shelf survey
+WJ20190709,,WJ20190709,,CP05MOAS-A6263-06-PARADN000,10,1,2019-07-10T12:41:00,2019-07-11T10:39:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50170,40.01075,-70.567783,,,,along-shelf survey
+AR-39,,AR-39,,CP05MOAS-A6263-00-ENG000000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGCON-AUVPNR-06263,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR-39,,AR-39,,CP05MOAS-A6263-01-FLORTN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-FLORTN-01108,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR-39,,AR-39,,CP05MOAS-A6263-02-DOSTAN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-DOSTAN-01519,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR-39,,AR-39,,CP05MOAS-A6263-03-CTDAVN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-CTDAVN-00424,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR-39,,AR-39,,CP05MOAS-A6263-04-NUTNRN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-NUTNRN-00366,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR-39,,AR-39,,CP05MOAS-A6263-05-ADCPAN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-ADCPAN-22076,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)
+AR-39,,AR-39,,CP05MOAS-A6263-06-PARADN000,11,1,2019-10-02T12:10:00,2019-10-02T22:36:00,CGVCP-05MOAS-00000,CGVEH-AUVPNR-06263,CGINS-PARADN-50170,40.205267,-71.0940167,,,,abbreviated mission due to time constraints (AL-1 to AL-2)

```

Comment:
Entire file reviewed and updated by @spetillo and @cdobs. Corrections made as we found issues.